gitversion.yml

```diff
@@ -1,14 +1,14 @@
 assembly-versioning-scheme: MajorMinorPatch
-mode: Mainline
-next-version: 1.46.0
+mode: ContinuousDeployment
```

Comment:
Maybe changes to this file should be in a different PR.

Reply:
This is part of the setup of the feature branch. Even if I agree that it would be nice to have it in separated branch, it still makes sense to be included in this PR as it's the first feature branch we built for Uno.